.github/workflows/cd.yml

```diff
@@ -2,7 +2,8 @@ name: cd
 
 on:
   push:
-    branches: [master]
+    tags:
+      - v*
```

Comment:
please use the release: [published] trigger instead of this. yes I know this one is in one repo, I probably gave you the wrong example, apologies for that.

Reply:
Done.